src/components/Tabs/Tabs.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import './Tabs.scss';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+class Tabs extends React.Component {
+  changeStatusHandler = (id) => {
+    const { idSelectedTab, onChangeSelTabs } = this.props;
```

Comment:
For this component you can create a function, not a class. It helps you make destructuring of props only one time. Now you make it here and into the render of this component